### PR TITLE
Tutorial explorer: move Sort By to a filter group from filter header

### PR DESCRIPTION
**Pull request by breville**
The Sort By dropdown is now moved out of the filter header and into a filter group.

screenshot 2017-10-17 22 05 20

### CODE REVIEW

**Comment by islemaster** [comment]
So... would you like me to merge my tests/ES6 PR before this?

File: apps/src/tutorialExplorer/filterGroupSortBy.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+const styles = {
+  select: {
+    width: '100%',
+    marginTop: 10,
+    height: 26
+  }
+};
+
+const FilterGroupSortBy = React.createClass({
```
Comment (islemaster) [inline apps/src/tutorialExplorer/filterGroupSortBy.jsx]: Please use ES6 classes for new React components.  [`React.createClass` is deprecated](https://reactjs.org/blog/2017/04/07/react-v15.5.0.html#new-deprecation-warnings) as of v15.5.0.

Let me know if you want any guidance on this.  Common gotchas include:
- The `getInitialState` function is no longer supported.
- The `isMounted` function is no longer supported.
- Component methods are not autobound, so you should express callbacks (like `handleChangeSort` here) as arrow functions.

**Comment by breville** [comment]
@islemaster Feel free to go ahead and merge the test & ES6 work as soon as you're ready, and I'll update to follow.

**Comment by islemaster** [comment]
:+1: Just merged it!

**Comment by breville** [comment]
Folks, quite a few updates made.

**Pull request merged**